test/parameterizer/processProposal.js

```diff
@@ -87,15 +87,15 @@ contract('Parameterizer', (accounts) => {
     });
 
     it('should not set new parameters if a proposal\'s processBy date has passed, ' +
-    'but should resolve any challenges against the domain', async () => {
+    'but challenge succeeded', async () => {
```

Comment:
Check the challenge.resolved flag is true at the end of the test. Put the original description back.